applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,114 @@
+You can even compile, flash and run the sample to verify proper operation of the modem.
+
+Flashing and running
+********************
+
+When built with the Zephyr-compatible modem configuration, SLM will put the nRF91 Series SiP to deep sleep when powered on.
+Zephyr's cellular modem driver running on the controlling chip will take care of waking up the nRF91 Series SiP, so it is advised to first flash SLM to the nRF91 Series SiP.
+
+However, before flashing the SLM built with the Zephyr-compatible modem configuration, make sure that the nRF91 Series modem has been set to the desired system mode.
+Do this by issuing an ``AT%XSYSTEMMODE`` command followed by an ``AT+CFUN=0`` command so that the modem saves the system mode to NVM.
+For example, to enable only LTE-M, issue the following command: ``AT%XSYSTEMMODE=0,1,0,0``
+You need to do this because the modem's system mode is not automatically set at any point, so the one already configured will be used.
+
+Additionally, if the controlling chip is an external MCU:
+
+* Make sure that the UART and the power pin are wired according to how they are configured in both the external MCU and the nRF91 Series SiP.
+
+Or if the controlling chip is the nRF52840 of the nRF9160 DK:
+
+* Make sure that the **PROG/DEBUG SW10** switch on the DK is set to **NRF91** when flashing SLM, and to **NRF52** when flashing the Zephyr application.
+  The switch also affects which chip is reset when the Reset button is pressed.
+* No wiring is needed as the routing between the pins happens internally.
+
+When the Zephyr application has been flashed to the controlling chip and starts up, things will go as follows:
+
+* If it has power management enabled (:kconfig:option:`CONFIG_PM_DEVICE` is equal to ``y``), when it powers on the modem (by calling ``pm_device_action_run(<dev>, PM_DEVICE_ACTION_RESUME)`` as the sample does) the cellular modem driver will start by toggling the modem's power pin to wake it up.
+* Otherwise, if power management is disabled (in which case SLM should have :ref:`CONFIG_SLM_START_SLEEP <CONFIG_SLM_START_SLEEP>` set to ``n`` and :ref:`CONFIG_SLM_POWER_PIN <CONFIG_SLM_POWER_PIN>` can be left undefined), the cellular modem driver will automatically proceed and expect the modem to already be started.
+* The cellular modem driver will start sending AT commands to SLM.
+  It will enable the network status notifications, gather some information from the modem, enable CMUX, and set the modem to normal mode (with an ``AT+CFUN=1`` command).
+  This will result in SLM' PPP starting automatically when the network registration is complete.
+* From this point onwards, assuming that the Zephyr application brought up the driver's network interface (as the sample does), it will be able to send and receive IP traffic through it.
```

Comment:
```suggestion
* From this point onwards, if the Zephyr application brought up the driver's network interface, it will be able to send and receive IP traffic through it.
  The :zephyr:code-sample:`cellular-modem` sample does this.
```